File: Service_eHospital_DongNai_A/Implement/me_LichHenChiTietService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class me_LichHenChiTietService : BaseService<me_LichHenChiTiet, Ime_LichHenChiTietRepository>
    , Ime_LichHenChiTietService
    {
    private readonly Ime_LichHenChiTietRepository _me_LichHenChiTietRepository;
    public me_LichHenChiTietService(Ime_LichHenChiTietRepository me_LichHenChiTietRepository) : base(me_LichHenChiTietRepository)
    {
    _me_LichHenChiTietRepository = me_LichHenChiTietRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/me_LichHenChiTiet.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { me_LichHenChiTiet } from './me_LichHenChiTiet.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class me_LichHenChiTietService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'LichHenChiTiet_id', 'LichHen_id', 'DichVu_id', 'NoiTao_id', 'NguoiTao_id', 'NgayTao', 'ThoiGianTao', 'GhiChu', 'MuiTiemSo', 'NoiCapNhat_id', 'NguoiCapNhat_id', 'NgayCapNhat'];

    List: me_LichHenChiTiet[] | undefined;
    ListFilter: me_LichHenChiTiet[] | undefined;
    FormData!: me_LichHenChiTiet;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "me_LichHenChiTiet";
    }
}

